### ISSUE TITLE
Border braking the grid

### ISSUE THREAD

**Issue by anstapol** [GramThanos/jsCalendar#31]
Hi, nice plugin guys. Thanks for your job!

Problem:
Alignment doesn't match when there are rows with and without selected days.

<img width="297" alt="screenshot 2019-01-02 at 11 48 14" src="https://user-images.githubusercontent.com/33395021/50588902-83219980-0e84-11e9-95e1-b72f839d76f7.png">

Also line height is broken when any day of the month is selected.

Solve:
Just in case somebody needs this, here is the fix:

```scss
.jsCalendar tbody td.jsCalendar-selected {
   border: 2px solid currentColor;
   line-height: 2.2rem;
   cursor: pointer;
   height: 36px;
   width: 36px;
}
```

p.s.: It would be nice to have access to the sass files, just to do easier colours & sizes overrides.

**Comment by GramThanos** [comment]
I was not able to reproduce the problem...
![image](https://user-images.githubusercontent.com/14858959/50592201-c5a2a080-0e9b-11e9-99f6-202df0252211.png)

So I took a wild guess and added the Bootstrap CSS... the problem came up.
This is not a bug, it is an incompatibility... bootstrap's [reboot](https://getbootstrap.com/docs/4.0/content/reboot/) sets `box-sizing: border-box;` which beaks the sizing.
So the best way to fix it is to set it to the default value `box-sizing: content-box;`.
(Patched 495a809d051ce4369419b1d884d7bfcafb0930c3)

It would be awesome if you can test it too.

Also, I would be happy to share the sass files but there are no sass files 😆 . I codded to library directly on CSS. If there were any, they would be on the source code. I may convert the CSS code to sass in the future.

**Comment by anstapol** [comment]
> I was not able to reproduce the problem...
> ![image](https://user-images.githubusercontent.com/14858959/50592201-c5a2a080-0e9b-11e9-99f6-202df0252211.png)
> 
> So I took a wild guess and added the Bootstrap CSS... the problem came up.
> This is not a bug, it is an incompatibility... bootstrap's [reboot](https://getbootstrap.com/docs/4.0/content/reboot/) sets `box-sizing: border-box;` which beaks the sizing.
> So the best way to fix it is to set it to the default value `box-sizing: content-box;`.
> (Patched [495a809](https://github.com/GramThanos/jsCalendar/commit/495a809d051ce4369419b1d884d7bfcafb0930c3))
> 
> It would be awesome if you can test it too.
> 
> Also, I would be happy to share the sass files but there are no sass files 😆 . I codded to library directly on CSS. If there were any, they would be on the source code. I may convert the CSS code to sass in the future.

ups 😅
I'm very sorry for my stupidness, using bootstrap reset was precisely the fault that caused my situation

**Comment by GramThanos** [comment]
Well I like bootstrap and it is quite popular, so it was a good thing that you found the incompatibility!

**Comment by GramThanos** [comment]
Compatibility fix included in v1.4.4
